qa/workunits/cephtool/test.sh

```diff
@@ -903,7 +903,7 @@ function test_mds_tell()
 
 function test_mon_mds()
 {
-  local FS_NAME=cephfs
+  local FS_NAME=testcephfs1
```

Comment:
ditto

Reply:
Rather, I'm not sure changing this file is all that useful in the spirit of the overall PR.

Reply:
https://github.com/ceph/ceph/pull/53526#discussion_r1372173346